File: bounce/src/main/java/is/ricko/bounce/model/BounceLink.java
package is.ricko.bounce.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.Date;

@Data
@Entity(name = "link")
@Table(name = "bounce_link")
@AllArgsConstructor
@NoArgsConstructor
public class BounceLink {
  @Column(name = "link_created")
  private Date created;
  @Column(name = "link_hits")
  private Integer hits;
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(name = "link_id")
  private Integer id;
  @Column(name = "link_name")
  private String name;
  @Column(name = "link_peeks")
  private Integer peeks;
  @Column(name = "link_title")
  private String title;
  @Column(name = "link_to")
  private String to;
}

File: bounce/src/main/kotlin/is/ricko/bounce/BounceLink.kt
package `is`.ricko.bounce

import java.util.*

data class BounceLink (
    val id: Int? = null,
    val name: String,
    val created: Date? = null,
    val hits: Int? = null,
    val peeks: Int? = null,
    val to: String,
    val title: String? = null
)
